### PR TITLE
Add bowtie to FusionInspector (resolves #7)

### LINKED ISSUE #7: Add bowtie to FusionInspector

### PR DESCRIPTION
**Pull request by jpfeil**
The latest version of Trinity (which is used in FusionInspector) requires bowtie.

resolves #7

### CODE REVIEW

File: mutation_calling/fusion/fusion-inspector/1.0.1/Dockerfile [hunk 1]
```diff
@@ -4,12 +4,13 @@
 # Run with sudo docker run -v <IO_folder>:/data aarjunrao/fusion-inspector:1.0.1 <parameters>
 
 # Use samtools
-FROM aarjunrao/samtools:1.3.1
+# This is temp until Arjun builds his version
+FROM jpfeil/samtools:1.3.1
```
Comment (arkal) [inline mutation_calling/fusion/fusion-inspector/1.0.1/Dockerfile]: this was built and put on the dockerhub before this PR was created!

**Review by arkal — CHANGES_REQUESTED** [review]

**Comment by arkal** [comment]
Tag me explicitly next time so i see this earlier

**Comment by jpfeil** [comment]
@arkal Sorry, Arjun! I removed that comment.

**Review by arkal — APPROVED** [review]

**Comment by arkal** [comment]
Uploaded to dockerhub

**Pull request merged**